### PR TITLE
Typos corrected

### PR DESCRIPTION
**Pull request by gmoraleda**
### Checklist

- [x] I've looked at the [contribution guidelines](https://github.com/raywenderlich/swift-algorithm-club/blob/master/.github/CONTRIBUTING.md).
- [x] This pull request is complete and ready for review.

### Description

Minor typos corrected.

### CODE REVIEW

**Comment by gmoraleda** [comment]
> I think your edit made the code more ambiguous

It's consistent with the rest of examples and the function content. The parameter name (`array`) and the signature didn't correspond to the function content nor the way it was being called.

File: Count Occurrences/README.markdown [hunk 1]
```diff
@@ -22,7 +22,7 @@ The trick is to use two binary searches, one to find where the `3`s start (the l
 In code this looks as follows:
 
 ```swift
-func countOccurrences<T: Comparable>(of key: T, in array: [T]) -> Int {
+func countOccurrences<T: Comparable>(of key: T, in a: [T]) -> Int {
```
Comment (kelvinlauKL) [inline Count Occurrences/README.markdown]: `array` would be preferable here. I would refer to some of the principles in the Swift API design [guidelines](https://swift.org/documentation/api-design-guidelines/) for this:

> Clarity is more important than brevity. Although Swift code can be compact, it is a non-goal to enable the smallest possible code with the fewest characters. Brevity in Swift code, where it occurs, is a side-effect of the strong type system and features that naturally reduce boilerplate.

Comment (kelvinlauKL) [reply Count Occurrences/README.markdown]: The README is not accurate with the actual code. All the `a` references should actually be `array`.

Comment (gmoraleda) [reply Count Occurrences/README.markdown]: Agree. Array is clearer. Just exchanged the renaming.

**Pull request merged**